chapter_preface/index_vn.md

```diff
@@ -22,7 +22,17 @@ Neural networks then, the antecedents of the deep learning models
 that we focus on in this book, were regarded as outmoded tools.
 -->
 
-*dịch đoạn phía trên*
+Chỉ một vài năm trước, không có nhiều nhà khoa học học sâu phát triển các sản
```

Comment:
Theo mình nên có giới thiệu kèm từ gốc tiếng Anh cho từ được đề cập lần đầu tiên, ví dụ
nhà khoa học làm về học sâu (Tiếng Anh: Deep Learning) 

Điều này giúp người đọc có cái nhìn liên kết với các tài liệu tiếng Anh khi có dịp đọc hay nghe người khác nói. Nó giúp giải quyết
1. sự không nhất quán hiện nay về cách dịch ở các sách khác nhau - người đọc có thể liên hệ ngay đến từ gốc
2. tránh tạo ra một tài liệu biệt lập với thế giới tiếng Anh nhất có thể.

Reply:
Mình đồng ý cần thêm kèm từ gốc tiếng Anh ở lần đầu tiên. Tuy nhiên chỉ cần viết từ tiếng Việt và mở ngoặc in nghiêng từ tiếng Anh là được. Ví dụ:

Machine learning (_học máy_).

Việc này cần thảo luận kỹ và làm hệ thống hơn ở một Pull Request khác. Trong PR này mình cứ viết chỉ tiếng Việt thôi cho nhất quán với những gì đã làm. Việc thay đổi cần được làm trên toàn bộ cuốn sách chứ không chỉ một phần nhỏ.